Pass bazel_startup_options also to the shutdown call to avoid it still using $HOME.
See https://github.com/google/jax/issues/7639

Author: Alexander Grund (TU Dresden)

diff --git a/build/build.py b/build/build.py
index dc275042..44a68a71 100755
--- a/build/build.py
+++ b/build/build.py
@@ -597,7 +597,7 @@ def main():
     f"--cpu={wheel_cpu}"])
   print(" ".join(command))
   shell(command)
-  shell([bazel_path, "shutdown"])
+  shell([bazel_path] + args.bazel_startup_options + ["shutdown"])
 
 
 if __name__ == "__main__":
